addons/csi/openstack/snapshot-controller.yaml

```diff
@@ -137,6 +137,9 @@ spec:
         app: snapshot-controller
     spec:
       serviceAccount: snapshot-controller
+      securityContext:
+        seccompProfile:
+          type: RuntimeDefault
```

Comment:
Tests are failing without this, so we need to add this. It's already present on 2.21 and master.